RecoDataProducts/inc/RecoProtonBunchIntensity.hh

```diff
@@ -0,0 +1,25 @@
+#ifndef RecoDataProducts_RecoProtonBunchIntensity_hh
+#define RecoDataProducts_RecoProtonBunchIntensity_hh
+#include <cmath>
+//
+// This object defines the reconstructed proton bunch intensity of a single microbunch (event).
+//
+// Original author M. MacKenzie, 2025
+namespace mu2e {
+  class RecoProtonBunchIntensity {
+    public:
+    explicit RecoProtonBunchIntensity(unsigned long long intensity = 0, unsigned long long uncertainty = 0) :
+      _intensity(intensity), _uncertainty(uncertainty) {}
+      unsigned long long intensity() const { return _intensity; }
+      unsigned long long uncertainty() const { return _uncertainty; }
+      bool operator == (RecoProtonBunchIntensity const& other ) const {
+        return _intensity == other.intensity() && _uncertainty == other.uncertainty(); }
+      bool operator != (RecoProtonBunchIntensity const& other ) const { return !(operator ==(other)); }
+    void add(RecoProtonBunchIntensity const& other) { _intensity += other.intensity(); _uncertainty = std::sqrt(std::pow(_uncertainty, 2) + std::pow(other.uncertainty(), 2)); }
```

Comment:
Why hard-code the uncertainty calculation?

Reply:
I was just adding the same `add` function as the MC PBI object. I could have it pass in a function that returns a new uncertainty given the two uncertainties to let the user define the method, would that be a better strategy? I could also remove this function and add setter functions, so a user could do the addition as defined by their analysis and just update the `_intensity` and `_uncertainty` fields